src/utils/email.test.ts

```diff
@@ -12,6 +12,25 @@ jest.mock("nodemailer", () => {
   };
 });
 
+jest.mock("../app/functions/server/logging", () => {
+  class Logging {
+    info(message: string, details: object) {
+      console.info(message, details);
+    }
+    warn(message: string, details: object) {
+      console.warn(message, details);
+    }
+    error(message: string, details: object) {
+      console.error(message, details);
+    }
+  }
+
+  const logger = new Logging();
+  return {
+    logger,
+  };
+});
```

Comment:
Nit, but this can be simplified a bit:

```suggestion
jest.mock("../app/functions/server/logging", () => {
  return {
    logger: {
      info: console.info,
      warn: console.warn,
      error: console.error,
    },
  };
});
```